service/resource/cloudformation/delete.go

```diff
@@ -3,14 +3,76 @@ package cloudformation
 import (
 	"context"
 
+	"github.com/aws/aws-sdk-go/aws"
+	awscloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
 )
 
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	deleteStackInput, err := toDeleteStackInput(deleteChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	stackName := deleteStackInput.StackName
+	if *stackName != "" {
+		_, err := r.awsClient.DeleteStack(&deleteStackInput)
+
+		if err != nil {
+			return microerror.Mask(err)
```

Comment:
It's not a standard yet but it would be nice to align the logging with the approach @kopiczko is using in azure-operator. WDYT?

https://github.com/giantswarm/azure-operator/blob/master/service/resource/resourcegroup/resource.go#L215

Reply:
Looks nice :) done